pkgs/by-name/be/beatsabermodmanager/package.nix

```diff
@@ -20,10 +20,7 @@ buildDotnetModule rec {
     fetchSubmodules = true; # It vendors BSIPA-Linux
   };
 
-  dotnet-sdk = with dotnetCorePackages; combinePackages [
-    sdk_7_0
-    sdk_6_0
-  ];
+  dotnet-sdk = dotnetCorePackages.sdk_7_0;
```

Comment:
Same, and https://github.com/NixOS/nixpkgs/pull/339370 will cause a merge conflict